.azure-pipelines/util/analyze-steps.yml

```
@@ -31,6 +31,16 @@ steps:
 
 - pwsh: 'Install-Module "platyPS", "PSScriptAnalyzer" -Force -Confirm:$false -Scope CurrentUser'
   displayName: 'Install PowerShell Dependencies'
+
+- task: PowerShell@2
+  displayName: 'Install latest modules'
+  inputs:
+    targetType: 'inline'
+    script: |
```

pwsh: true is required because otherwise it will try to use windows powershell
```suggestion
    pwsh: true
    script: |
```